src/routes/tools/local-climate-change-snapshot/_data.js

```diff
@@ -132,10 +135,15 @@ const createRanges = (_data) => {
  * @param {string} method - default is GET, POST for uploaded boundaries
  * @return {array}
  */
-export async function getObserved(config, params, method = "GET") {
+export async function getObserved(
+  config,
+  params,
+  method = "GET",
```

Comment:
if the `method` param is the least likely to be modified I might make that the last param as it's optional. Or wrap the params in an object.

Reply:
Don't forget to update the doc string comment too with the param for `searchStr`

Reply:
Fixed in https://github.com/berkeley-gif/caladapt-website-2021/pull/209/commits/a0404de4c1dba0b67f5b07113c31c051e6c2f015